[bugfix] QA and bump CentOS box version

trombik opened the pull request.

Commit:
QA

Commit:
[bugfix] QA and bump CentOS box version

Commit:
[bugfix] s/centos-7.2-x86_64/centos-7.3-x86_64/g

mitsururike commented on README.md:
```diff
@@ -1,89 +1,78 @@
+| `monit_group` | group of `monit` | `{{ __monit_group }}` |
+| `monit_service` | service name of `monit` | `monit` |
+| `monit_conf_dir` | directory name where `monitrc` is | `{{ __monit_conf_dir }}` |
+| `monit_conf_file` | path to `monitrc` | `{{ __monit_conf_dir }}/monitrc` |
```
common error: s/__monit_conf_dir/monit_conf_dir/

mitsururike commented on tasks/main.yml:
```diff
@@ -31,13 +31,15 @@
   notify: Restart monit
 
-- shell: "find {{ monit_conf_dir }} -iname '{{ monit_conf_file }}.*@*~' -type f | sort | head -n 3 | xargs -r -n 1 rm"
+- name: Find old monit.conf
```
the name is not appropriate as it removes the old file, not just finds it.

Commit:
[bugfix] Rename the task

https://github.com/reallyenglish/ansible-role-monit/pull/15#discussion_r133347851

Commit:
[bugfix] common error: s/__monit_conf_dir/monit_conf_dir/

https://github.com/reallyenglish/ansible-role-monit/pull/15#pullrequestreview-56517804
